docs/release-notes/next/PT-165413555_FSM_GA_awareness.md

```diff
@@ -0,0 +1,8 @@
+* Introduces State Channel FSM Generalized Accounts awareness. It now fully
+  supports creation of meta transactions and verifies them.
+* Changes WebSocket API regarding signing transactions: now sign requests
```

Comment:
> * the `FSM_B` sends the co-authenticated tx to `FSM_B` (3)
> 
> Note that `3` changes the `noise` protocol message and expectations for the message format itself, thus making it non-backwards compatible with older versions.

Shall there be somewhere in this file at least a mention of the channels Noise node-to-node protocol breakage/change?